account.go

```diff
@@ -1,22 +1,267 @@
 	Timezone         string   `json:"timezone"`
+	BusinessName     string   `json:"business_name"`
+	SupportPhone     string   `json:"support_phone"`
+	ProductDesc      string   `json:"product_description"`
+	Managed          bool     `json:"managed"`
+	DebitNegativeBal bool     `json:"debit_negative_balances"`
+	Keys             *struct {
+		Secret  string `json:"secret"`
+		Publish string `json:"publishable"`
+	} `json:"keys"`
+	Verification *struct {
+		Fields    []string `json:"fields_needed"`
+		Due       *int64   `json:"due_by"`
+		Contacted bool     `json:"contacted"`
+	} `json:"verification"`
+	LegalEntity      *LegalEntity      `json:"legal_entity"`
+	TransferSchedule *TransferSchedule `json:"transfer_schedule"`
+}
+
+// LegalEntity is the structure for properties related to an account's legal state.
+type LegalEntity struct {
+	Type             LegalEntityType      `json:"type"`
+	BusinessName     string               `json:"business_name"`
+	Address          Address              `json:"address"`
+	First            string               `json:"first_name"`
+	Last             string               `json:"last_name"`
+	PersonalAddress  Address              `json:"personal_address"`
+	DOB              DOB                  `json:"dob"`
+	AdditionalOwners []Owner              `json:"additional_owners"`
+	Verification     IdentityVerification `json:"verification"`
+	SSN              string               `json:"ssn_last_4"`
+	PersonalID       string               `json:"personal_id_number"`
+	BusinessTaxID    string               `json:"business_tax_id"`
+	BusinessVatID    string               `json:"business_vat_id"`
+}
+
+// Address is the structure for an account address.
+type Address struct {
+	Line1   string `json:"line1"`
+	Line2   string `json:"line2"`
+	City    string `json:"city"`
+	State   string `json:"state"`
+	Zip     string `json:"postal_code"`
+	Country string `json:"country"`
+}
+
+// DOB is a structure for an account owner's date of birth.
+type DOB struct {
+	Day   int `json:"day"`
+	Month int `json:"month"`
+	Year  int `json:"year"`
+}
+
+// Owner is the structure for an account owner.
+type Owner struct {
+	First        string               `json:"first_name"`
+	Last         string               `json:"last_name"`
+	DOB          DOB                  `json:"dob"`
+	Address      Address              `json:"address"`
+	Verification IdentityVerification `json:"verification"`
+}
+
+// IdentityVerification is the structure for an account's verification.
+type IdentityVerification struct {
+	Status   IdentityVerificationStatus `json:"status"`
+	Document *struct {
+		ID      string `json:"id"`
+		Created int64  `json:"created"`
+		Size    int64  `json:"size"`
+	} `json:"document"`
+	Details *string `json:"details"`
+}
+
+// TransferSchedule is the structure for an account's transfer schedule.
+type TransferSchedule struct {
+	Delay    int      `json:"delay_days"`
+	Interval Interval `json:"interval"`
```

Comment:
OK, since the types were different (string vs. int), I added them as two separate properties.

Just to be clear, the TransferSchedule is just gettable on an account, or can it also be set via the API?